<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>460</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>417</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>421</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <widget class="QLabel" name="lblHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>411</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/Hearts&amp;stars.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="wPickSpotButtons" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>250</y>
       <width>201</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(64, 190, 251);
background-image: url(:/images/board.png);</string>
     </property>
     <widget class="QPushButton" name="btn10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn01">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn02">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn00">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn21">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>140</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn22">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn12">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn20">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border: 0px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lblStatus">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(64, 190, 251);
color: rgb(255, 211, 60);</string>
     </property>
     <property name="text">
      <string>Hearts WIN!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartOver">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>190</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 57, 140);</string>
     </property>
     <property name="text">
      <string>START OVER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>160</y>
       <width>75</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(231, 57, 140);</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblText1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(64, 190, 251);</string>
     </property>
     <property name="text">
      <string>A remake of the classic 'Tic-Tac-Toe', using hearts and stars.</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblText2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>401</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Rounded MT Bold</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(64, 190, 251);</string>
     </property>
     <property name="text">
      <string>There are two players. Player 1, pick your symbol and press START.</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>160</y>
       <width>218</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="hlaySymbolSelect">
      <item>
       <widget class="QLabel" name="lblP1Symbol">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>13</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(64, 190, 251);</string>
        </property>
        <property name="text">
         <string>Player 1 Symbol </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbSymbols">
        <property name="font">
         <font>
          <family>Arial Rounded MT Bold</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(64, 190, 251);</string>
        </property>
        <item>
         <property name="text">
          <string>Hearts</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Stars</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
